BinaryObjectScanner/Protection/CopyX.cs

```diff
@@ -114,22 +116,33 @@ public ConcurrentQueue<string> CheckDirectoryPath(string path, IEnumerable<strin
             // Excludes files with .x64 extension to avoid flagging Professional files.
             // Sorts list of files in ZDAT* so just the first file gets pulled, later ones have a chance of the ring 
             // intersecting the start of the file.
-            var fileList = files.Where(f => !f.EndsWith(".x64", StringComparison.OrdinalIgnoreCase))
-                .Where(f =>
+
+            string[] dirs = ["ZDAT", "System/", "System\\"];//Kenny's Adventure uses System instead of ZDAT.
```

Comment:
The two variants of `System` are unnecessary. If you really want both, you should have a single one that uses `Path.DirectorySeparator` appended to the end, which will be OS-appropriate.